src/Service.Host/client/src/containers/parts/PartTemplates.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from 'react-redux';
+import { initialiseOnMount } from '@linn-it/linn-form-components-library';
+import partTemplatesActions from '../../actions/partTemplatesActions';
+import partTemplatesSelectors from '../../selectors/partTemplatesSelectors';
+import partTemplateSelectors from '../../selectors/partTemplateSelectors';
+import PartTemplatesSearch from '../../components/parts/PartTemplatesSearch';
+import { getPrivileges } from '../../selectors/userSelectors';
+
+const mapStateToProps = (state, { match }) => ({
+    items: partTemplatesSelectors.getItems(state),
+    loading: partTemplatesSelectors.getLoading(state),
+    linkToSources: match?.url?.endsWith('/sources'),
+    privileges: getPrivileges(state),
+    partTemplates: partTemplatesSelectors.getItems(state),
+    partTemplate: partTemplateSelectors.getItem(state)
+});
+
+const initialise = () => dispatch => {
+    dispatch(partTemplatesActions.fetch());
+};
+
+const mapDispatchToProps = {
+    initialise,
+    fetchItems: partTemplatesActions.fetchByQueryString,
```

Comment:
was fetchItems used? Seems  you already fetched items in the initialise() so no need to pass it to the component as a prop I'd imagine